src/h5web/vis-packs/core/visualizations.ts

```diff
@@ -17,19 +17,21 @@ import {
   MatrixVisContainer,
   LineVisContainer,
   HeatmapVisContainer,
-  PhaseMapVisContainer,
+  ComplexVisContainer,
 } from './containers';
 import type { VisDef } from '../models';
 import { LineConfigProvider } from './line/config';
 import { HeatmapConfigProvider } from './heatmap/config';
+import ComplexToolbar from '../../toolbar/ComplexToolbar';
+import { ComplexConfigProvider } from './complex/config';
 
 export enum Vis {
   Raw = 'Raw',
   Scalar = 'Scalar',
   Matrix = 'Matrix',
   Line = 'Line',
   Heatmap = 'Heatmap',
-  PhaseMap = 'Phase map',
+  Complex = 'Complex',
```

Comment:
I am not awfully happy with the name **Complex** so I am open to suggestions

Reply:
I don't mind it, but we'll ask Thomas and Andy on Tuesday 😉